test/executors.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+. ./store.sh
+
+function exec_conjur_client() {
+  kubectl --namespace "$(store_get TEST_NAMESPACE)" \
```

Comment:
Maybe another crazy idea... just throwing this out there like throwing spaghetti against the wall :)  ... Would it be possible to keep the original  `$VARIABLE_NAME` environmental variable format in these test bash scripts, and instead create a summon-like program that reads a secrets.yml and retrieves environment from your key-value store (rather than a traditional secrets provider)?

Reply:
@diverdane My response to this is why would you want to use the environment variable format ? I'd understand if `exec_conjur_client` was being consumed in some place that didn't use the store. If that was the case I'd opt for variable precedence where environment variables can take priority over the store value e.g. 
```
{VARIABLE:-$(store_get VARIABLE)}
```

Reply:
Though I'm trying to steer clear of environment variables because they're implicit and it's cumbersome to debug for their values once the process is done.

Reply:
I see your point, thanks for the well-thought-out explanation. In the bash scripts, it's clearer to have the actual store_get/store_set mechanisms called out directly, rather than being abstracted through a summon-like translation layer.